packages/search/lib/commands/index.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { strict as assert } from 'node:assert';
-import { pushArgumentsWithLength, pushSortByArguments } from '.';
+
+/* import { pushArgumentsWithLength, pushSortByArguments } from '.';
```

Comment:
have a feeling this can just be deleted, as functions being tested no longer exist.